source/code/plugin/health/health_model_buffer.rb

```diff
@@ -0,0 +1,29 @@
+module HealthModel
+
+=begin
+    Class that is used to create a buffer for collecting the health records
+=end
+    class HealthModelBuffer
+
+        attr_reader :records_buffer, :log
+
+        def initialize
+            @records_buffer = []
+        end
+
+        # Returns the current buffer
+        def get_buffer
+            return @records_buffer
+        end
+
+        # adds records to the buffer
+        def add_to_buffer(records)
```

Comment:
we don't cleanup buffer ? Does it always grow on a cluster ?

Reply:
yes, we do. Every minute. 
in the model builder plugin